openedx/core/djangoapps/content/learning_sequences/data.py

```diff
@@ -241,6 +258,14 @@ class ScheduleData:
     sequences = attr.ib(type=Dict[UsageKey, ScheduleItemData])
 
 
+@attr.s(frozen=True)
+class SpecialExamAttemptData:
+    """
+    Overall special exam attempt data.
+    """
+    sequences = attr.ib(type=Dict[UsageKey, Dict])
```

Comment:
Can you describe what exactly is stored in this field?

Reply:
This is an example of the object I receive back:
```
{
    "suggested_icon": "fa-clock-o",
    "in_completed_state": false,
    "short_description": "Timed Exam"
}
```

I don't know for certain whether there are additional fields that can be included and/or whether these fields are always present. I'll be looking into the Proctoring API for this info